docs/rl-algorithms/td3.md

```diff
@@ -0,0 +1,109 @@
+# Twin Delayed Deep Deterministic Policy Gradient (TD3)
+
+
+## Overview
+
+TD3 is a popular DRL algorithm for continuous control. It extends DQN to work with the continuous action space by introducing a deterministirc actor that directly outputs continuous actions. DDPG also combines techniques from DQN such as the replay buffer and target network.
```

Comment:
I would say it is more an extension of DDPG than DQN, which adds 1) two Q-networks instead of one to compensate the overstimation bias issue, 2) delays the updates of the policy, 3) uses a "target policy" to compute the Bellman target ...

Reply:
Fixed.